### ISSUE TITLE
Getting Spock back on Stackage

### ISSUE THREAD

**Issue by Zialus** [agrafix/Spock#170]
Spock has been on a LTS Stackage release since 12.26 - https://www.stackage.org/lts-12.16/package/Spock-0.13.0.0

Are there any plans to add it back?

**Comment by agrafix** [comment]
Yes, I was planning to add it back but haven't gotten around to submitting a PR to the repo yet. Feel free to go for it if you are interested!

**Comment by Zialus** [comment]
Any news here? What would be the steps do it?

**Comment by Zialus** [comment]
@jberryman Are you familiar with the process to make this happen?

**Comment by jberryman** [comment]
@Zialus I'm not totally sure…I don't have bandwidth to be the stackage maintainer, but maybe another user can step up?

**Comment by Zialus** [comment]
Looks like it's just a matter of editing a yaml in the stackage repo. https://github.com/commercialhaskell/stackage#add-your-package

It might be important to fix this first though:  https://github.com/agrafix/Spock/issues/180
